src/components/horizontal-fact-card.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import Img from "gatsby-image"
+import { makeStyles } from '@material-ui/core/styles';
+import Card from '@material-ui/core/Card';
+import { CardActionArea } from 'gatsby-theme-material-ui'
+import CardMedia from '@material-ui/core/CardMedia';
+import CardContent from '@material-ui/core/CardContent';
+import Typography from '@material-ui/core/Typography';
+
+const useStyles = makeStyles({
+    relatedCard: {
+        marginBottom: '5px',
```

Comment:
This margin should be handled in the component using it and should be more like `16px`